tuxemon/graphics.py

```diff
@@ -31,8 +31,6 @@
 
 
 ColorLike = Union[
-    pygame.color.Color,
```

Comment:
Why removing `pygame.color.Color` as valid alias for a color?

I feel like pygame has some nice named constants for color.

Reply:
Because pygame.color.Color accepts tuples too, but I can leave it, no issue